yt_dlp/extractor/ard.py

```diff
@@ -577,3 +578,117 @@ def fetch_page(page_num):
         return self.playlist_result(
             OnDemandPagedList(fetch_page, self._PAGE_SIZE), full_id, display_id=display_id,
             title=page_data.get('title'), description=page_data.get('synopsis'))
+
+
+class ARDAudiothekIE(InfoExtractor):
+    IE_NAME = 'ARD:audiothek'
+    _VALID_URL = r'''(?x)https://
+        (?:www\.)?ardaudiothek\.de/
+        (?:player|live|episode|(?P<playlist>sendung|serie|sammlung))/
+        (?P<display_id>(?(playlist)[^?#]+?|[^?#]+))/
+        (?P<id>[a-zA-Z0-9]+)
+        (?(playlist)/(?P<season>\d+)?/?(?:[?#]|$))'''
+
+    _TESTS = [{
+        'url': 'https://www.ardaudiothek.de/sendung/1live-caiman-club/53375276/',
+        'info_dict': {
+            'id': '53375276',
+            'title': '1LIVE Caiman Club',
+            'description': 'md5:003cff043a41b14cf045b960b89aaa86',
+        },
+        'playlist_mincount': 22,
+    }, {
+        'url': 'https://www.ardaudiothek.de/episode/1live-caiman-club/caiman-club-s04e04-cash-out/1live/13556081/',
+        'info_dict': {
+            'id': '13556081',
+            'ext': 'mp3',
+            'upload_date': '20240717',
+            'duration': 3339,
+            'title': 'CAIMAN CLUB (S04E04): Cash Out',
+            'thumbnail': 'https://api.ardmediathek.de/image-service/images/urn:ard:image:d5014b612429c396',
+            'description': 'md5:8decf7974ed1cbf5a9d2c537940e1c4b',
+            'display_id': '1live-caiman-club/caiman-club-s04e04-cash-out/1live',
+            'timestamp': 1721181641,
+            'series': '1LIVE Caiman Club',
+
+        },
+    }]
+
+    _QUERY_PLAYLIST = '''show(id: "%s") {
```

Comment:
For consistency with below
```suggestion
    _QUERY_PLAYLIST = '''\
    show(id: "%s") {
```